docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,462 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+Access Monitoring allows users to understand and analyze the access patterns in a Teleport cluster.
+
+
+When Access Monitoring is enabled in a cluster, Teleport runs a background service that collects
+data from different input sources, builds graphical reports highlighting usage patterns
+within the cluster, and allows users to create automated response actions.
+
+The primary data source for Access Monitoring is the Teleport audit log. Out of the box,
+Teleport includes a report that searches the audit log for events indicating risky usage patterns (such as SSH
+or database sessions without MFA) and provides users with recommendations on suggested actions.
+
+Users are able to write their own custom access monitoring queries by querying the audit log.
+
+## Prerequisites
+- Teleport v14 or later.
+- For Self-Hosted Teleport the [AWS Athena Backend](../reference/backends.mdx#athena) is required.
+
+
+### Configuration
+
+<Tabs>
+<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
+
+To enable Access Monitoring you need to update the Teleport Auth Service configuration:
+```yaml
+auth_service:
+  access_monitoring:
+    enabled: true
+    # AWS role ARN that Teleport will assume to execute Athena SQL queries.
+    role_arn: arn:aws:iam::111111111111:role/access-monitoring-role
+    # S3 bucket where Access Monitoring reports will be stored.
+    report_results: s3://audit-log-example/report_results
```

Comment:
Something that just occurred to me, we should also tell users what IAM permissions Teleport needs to use access monitoring, right? We can add it as a follow up.

Reply:
Right, I will do this as fallow up.